grails-app/views/invoice/_invoiceItems.gsp

```diff
@@ -18,7 +18,7 @@
             </tr>
             </thead>
             <tbody>
-            <g:each var="invoiceItem" in="${invoiceInstance.invoiceItems.sort { it.dateCreated }}" status="status">
+            <g:each var="invoiceItem" in="${invoiceInstance.getSortedInvoiceItems()}" status="status">
```

Comment:
nitpicky, but you've always said you liked the shortened groovy versions, so: you could use the `invoiceInstance.sortedInvoiceItems` here :smile: